src/concepts/ecosystem.md

```diff
@@ -0,0 +1,24 @@
+# DataJoint Ecosystem 
+
+An overview of the DataJoint ecosystem is provided below: 
+
+### Open Source softwares
```

Comment:
Does this list overlap with [this page](https://github.com/datajoint/element-deeplabcut/commit/352c44f98325cb79a249026d84310bff8417e7e2)

Reply:
Thanks @CBroz1.  I think this is the wrong link.

Reply:
Oops, my bad - I meant the [core list page](https://datajoint.com/docs/core/)

Reply:
This is correct. We won't need the `ecosystem` page